packages/loot-design/src/components/sidebar.js

```diff
@@ -198,6 +198,59 @@ function SecondaryItem({
   );
 }
 
+export function ChartItem({
```

Comment:
Why is this in `sidebar.js`?

Reply:
Sorry I copied the format of SecondaryItem which is in sidebar and forgot to move it. I've moved it to Charts.js